obj/login.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    //DOM elements
+    userSelect : ".Select-placeholder",
+    userSelectedValue : ".Select-value-label",
+    userSelectListValue : '[aria-label="?"]',
+    roleSelect : '[id="react-select-3--value"]',
+    roleSelectListValue : '[aria-label="?"]',
+    roleSelectedValue : '[id="react-select-3--value"]',
```

Comment:
This seems to be a same locator as previous one, Page object suggest to try not use duplicate locators, ether refactor it and use same one, or try to make it unique